crates/sui-framework/packages/deepbook/sources/clob_v2.move

```diff
@@ -185,6 +185,27 @@ module deepbook::clob_v2 {
         /// owner ID of the `AccountCap` that withdrew the asset
         owner: address
     }
+
+    /// Returned as metadata only when a maker order is filled from place order functions.
+    struct MatchedOrderMetadata<phantom BaseAsset, phantom QuoteAsset> has copy, store, drop {
```

Comment:
is copy really needed?

Reply:
copy ability is not required as such. But since this struct represents only meta data copy ability could be useful for third party protocols which might want to update several entities based on this. Hence copy ability provides easier replication of meta data objects as per requirement.